core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineStrategy.java

```diff
@@ -33,40 +31,34 @@ public interface RdbEngineStrategy {
 
   String computeBooleanValue(boolean value);
 
-  void createNamespaceExecute(Connection connection, String fullNamespace) throws SQLException;
+  String[] createNamespaceExecuteSqls(String fullNamespace);
```

Comment:
Maybe we don't need `Execute`?
```suggestion
  String[] createNamespaceSqls(String fullNamespace);
```

Reply:
🙆‍♀️ 
2629af5
b9e3c876